generated/nifake/tests/test_session.py

```diff
@@ -565,3 +565,21 @@ def test_set_vi_string_attribute(self):
         with nifake.Session('dev1') as session:
             session.read_write_string = attrib_string
             self.patched_library.niFake_SetAttributeViString.assert_called_once_with(SESSION_NUM_FOR_TEST, b'', attribute_id, b'This is test string')
+
+    def test_multipoint_read(self):
+        test_maximum_time = 1000
+        test_reading_array = [1.0, 0.1, 42, .42]
+        test_actual_number_of_points = len(test_reading_array)
+        self.patched_library.niFake_ReadMultiPoint.side_effect = self.side_effects_helper.niFake_ReadMultiPoint
+        self.side_effects_helper['ReadMultiPoint']['readingArray'] = test_reading_array
+        self.side_effects_helper['ReadMultiPoint']['actualNumberOfPoints'] = test_actual_number_of_points
+        with nifake.Session('dev1') as session:
+            measurements, points = session.read_multi_point(test_maximum_time, len(test_reading_array))
+            assert len(measurements) == test_actual_number_of_points
+            assert points == test_actual_number_of_points
+            assert measurements == test_reading_array
+            from mock import call
+            calls = [call(SESSION_NUM_FOR_TEST, test_maximum_time, len(test_reading_array), ANY, ANY)]
+            self.patched_library.niFake_ReadMultiPoint.assert_has_calls(calls)
+            assert self.patched_library.niFake_ReadMultiPoint.call_count == 1
```

Comment:
FYI
You can write lines 581 through 584 as a single  line

    self.patched_library.niFake_ReadMultiPoint.assert_called_once_with(SESSION_NUM_FOR_TEST, test_maximum_time, len(test_reading_array), ANY, ANY)